requirements-test-libraries.txt

```diff
@@ -76,6 +76,7 @@ web3==5.25.0
 websockets==10.1; python_version >= '3.7'
 zeep==4.1.0
 pypsexec==0.3.0
+mimesis==5.2.1

```

Comment:
It seems that `mimesis` has dropped support for both Python 3.6 and 3.7 so this will need a python version constraint (like `websockets` has).